### PR TITLE
DBZ-1972 move static Filed methods as static Filed constant to RelationalDatabaseConnectorConfig instead.

### PR DESCRIPTION
**Pull request by JanHendrikDolling**
https://issues.jboss.org/browse/DBZ-1972

Instead of deleting the static methods, I can alternatively mark them as deprecated. WDYT?

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java [hunk 1]
```diff
@@ -269,6 +270,16 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withDescription("A comma-separated list of regular expressions matching fully-qualified names of columns that should "
                     + "be masked by hashing the input. Using the specified hash algorithms and salt.");
 
+    public static final Field MASK_COLUMN = Field.create("column.mask.with.(d+).chars")
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java:273]: Have you checked whether/how that placeholder with d+ appraoch works with Kafka Connect? I.e. is there perhaps a warning about an unknown field being given in the logs? But then perhaps that was the case already before?

**Comment by gunnarmorling** [comment]
> Instead of deleting the static methods, I can alternatively mark them as deprecated. WDYT?

No need to do so, they were only used in our own tests and were not meant for public consumption.

**Comment by gunnarmorling** [comment]
Could you also rebase this PR to the latest master? Thanks!

File: debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java (hunk 1 above)
Comment (JanHendrikDolling) [reply debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java:273]: I looked at the Kafka and the connector log. 
Only the connector log contains information.
But it's only the normal startup parameter info form the connector configuration.

So I don't think there is a warning about an unknown field. But maybe I have only looked into the wrong log file.

Comment (gunnarmorling) [reply debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java:287]: Ok, thanks!

**Comment by gunnarmorling** [comment]
Rebased and applied. Thanks, @JanHendrikDolling!

**Pull request closed without merging**